modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISCompound3DTestSetup.java

```diff
@@ -132,4 +135,12 @@ protected void dropPointCompound3DTable() throws Exception {
         run("DELETE FROM  GEOMETRY_COLUMNS WHERE F_TABLE_NAME = 'pointCompound3d'");
         run("DROP TABLE \"pointCompound3d\"");
     }
+
+    @Override
+    protected void setUpDataStore(JDBCDataStore dataStore) {
```

Comment:
Would be best if the tests run in both cases. Could you add a new test that enables estimated extents in a try/finally block instead?